### ISSUE TITLE
bug : the link between the charity and coordinator isn't symmetrical - How to update investigate

### ISSUE THREAD

**Issue by RachWalm** [RachWalm/VolunteerVillage#101]
### Story

As a **Coordinator**, I can **assign a charity or coordinator to each other only once** so that I can **avoid repeating myself**.

### Acceptance Criteria

- Coordinator and charity link works symmetrically or at least efficiently.

### Tasks

- [ ] look at why symmetrical isn't working on the many to many - does it just have to be the name?
- [ ] find an efficient way for the charity and coordinator link to be updated.
- [ ] Do a charity/coordinator update page template/url/view

**Comment by RachWalm** [comment]
removed symetrical from the model and put in related names, 
Used set to save to the otherside of the relationship and saved it in stages with the cleaned form data.
Decided that an update charity/coordinator page would be a nice to have rather than to be included in this so decided to not do that part as it can be updated from the charity update page.
